dialogue/TownNPCs/ShovelGuy/ShovelGuy3.json

```diff
@@ -12,8 +12,8 @@
       "I'd like to dig out some space and lay the ground works for a fountain in the future!"
     ],
     "pl": [
-      "Ciekawe... jeśli zrobię dobrą robotę, możę będe mógł pracować na więcej projektach miasteczkowych w przyszłości?",
-      "To nie wygląda jak najfajniejsza robota, ale wszyscy musimy gdziesz zacząć, nie?",
+      "Ciekawe... jeśli zrobię dobrą robotę, możę będę mógł pracować na więcej projektach miasteczkowych w przyszłości?",
+      "To nie wygląda jak najfajniejsza robota, ale wszyscy musimy zacząć gdziesz, nie?",
```

Comment:
Change "gdziesz" to "gdzieś". I'd also suggest writing "gdzieś" before "zacząć"

Reply:
I intentionally put "gdzieś" after "zacząć" as I thought it would sound more emphasised that way.